spyder/plugins/configdialog.py

```diff
@@ -317,7 +317,12 @@ def load_from_conf(self):
             radiobutton.toggled.connect(lambda _foo, opt=option:
                                         self.has_been_modified(opt))
         for lineedit, (option, default) in list(self.lineedits.items()):
-            lineedit.setText(self.get_option(option, default))
+            option_value = self.get_option(option, default)
+            if isinstance(option_value, tuple):

```

Comment:
`list` for `EdgeLine` class and `tuple` for configuration

I think Its better to use list for `columns` attribute of `EdgeLine` class; and when loading data from configuration, `get_option` will return a tuple when reading comma separated string (e.g. `72,80`)